server/src/repositories/person.repository.ts

```diff
@@ -272,16 +328,18 @@ export class PersonRepository {
     return Number(result.numChangedRows ?? 0);
   }
 
-  getById(personId: string): Promise<PersonEntity | null> {
-    return (this.db //
-      .selectFrom('person')
-      .selectAll('person')
-      .where('person.id', '=', personId)
-      .executeTakeFirst() ?? null) as Promise<PersonEntity | null>;
+  getById(personId: string) {
+    return (
+      this.db //
+        .selectFrom('person')
+        .selectAll('person')
+        .where('person.id', '=', personId)
+        .executeTakeFirst() ?? null
```

Comment:
this is saying `return a promise or return null`, not `return a promise of undefined or null`. Just remove `?? null`